chatkit-core/config/baseline.xml

```diff
@@ -2,6 +2,9 @@
 <SmellBaseline>
   <Blacklist></Blacklist>
   <Whitelist>
+    <ID>ForbiddenComment:ChatkitTokenProvider.kt$// TODO: needs more refinements, essentially userId, authData -&gt; queryParameters, headers</ID>
+    <ID>ForbiddenComment:PlatformClient.kt$PlatformClient$// TODO: make internal, there are problems with Beams code that exposes it that need to be addressed</ID>
+    <ID>ForbiddenComment:UserSubscriber.kt$UserSubscriber$// TODO: handle onEnd, e.g. when tackling connection status (terminal/non-terminal errs)</ID>
```

Comment:
LOL

Reply:
We need to make sure each of these todo's are written into our discovery doc so the work needed is not lost.

Reply:
I've made sure everything is explicitly mentioned in the discovery doc (expanding on some points or changing their order).